firestore-bigquery-export/extension.yaml

```diff
@@ -193,6 +193,14 @@ params:
     default: posts
     required: true
 
+  - param: EXT_INSTANCE_ID
+    label: Extension Instance ID
+    description: >-
+      What is the unique identifier for this instance of the extension? This ID
+      ensures that multiple instances do not interfere with one another. It is
+      set automatically by Firebase but can also be customized if required.
+    required: true
+
```

Comment:
```suggestion
```
Since other extensions don't use unique extension ID for the events, let's keep it consistent.

Reply:
I added this based on your previous comments regarding several instances of the same extension.